test/OpenSpaceToolkit/Astrodynamics/Trajectory/State.test.cpp

```diff
@@ -362,15 +397,6 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_State, StreamOperator)
 
 TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_State, IsDefined)
```

Comment:
Add test with `nullptr` Frame and undefined Frame?

Reply:
@Derollez you happy with this one too?

Reply:
Resolving